core/field_dropdown.js

```diff
@@ -421,15 +430,18 @@ Blockly.FieldDropdown.prototype.isOptionListDynamic = function() {
 
 /**
  * Return a list of the options for this dropdown.
+ * @param {boolean=} opt_regenerate Whether or not to re-generate the options.
  * @return {!Array.<!Array>} Array of option tuples:
  *     (human-readable text or image, language-neutral name).
  * @throws If generated options are incorrectly structured.
  */
-Blockly.FieldDropdown.prototype.getOptions = function() {
+Blockly.FieldDropdown.prototype.getOptions = function(opt_regenerate) {
```

Comment:
`opt_regenerate` probably needs to default to true for backwards compat.

Reply:
Yeah that makes sense.